packages/overlay/src/vaadin-overlay-mixin.js

```diff
@@ -493,7 +501,6 @@ export const OverlayMixin = (superClass) =>
       }
 
       const evt = new CustomEvent('vaadin-overlay-outside-click', {
-        bubbles: true,
```

Comment:
Technically not necessary to remove this at the moment as the event would not go through the shadow root. But I figured it would be safer to remove this in case the DOM structure changes again at some point.